ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.12.yaml

```diff
@@ -138,7 +138,7 @@ tests:
       UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.12 and @aws-ipi and not @fips'
     test:
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi-network-mtu-localzone
+    workflow: cucushift-installer-rehearse-aws-ipi-network-mtu-edge-zone
 - as: aws-ipi-network-mtu-localzone-sdn-f28
```

Comment:
Shall we update the test name to reflect the workflow name change ?

Reply:
I guess that is because the install workflow's default value for `EDGE_ZONE_TYPES` is `localzone`

Reply:
there are several types for `edge-zone`, `localzone` is the default type.

Reply:
Ack.
Thanks for the clarification.

Reply:
Yes, `*edge-zone*` workflow supports `local-zone` and `wavelength-zone`